crates/bitcoind_rpc/tests/test_emitter.rs

```diff
@@ -18,7 +18,7 @@ use bitcoincore_rpc::RpcApi;
 /// 3. Reorg highest 6 blocks.
 /// 4. Emit blocks from [`Emitter`] and re-update the [`LocalChain`].
 #[test]
-pub fn test_sync_local_chain() -> anyhow::Result<()> {
+pub fn test_sync_local_chain() -> Result<()> {
```

Comment:
Can we still leave this as `anyhow::Result`? I don't want to get this confused with the `Result` in `core`.

Reply:
Did you and @evanlinjin agree to keep this as `Result<()>` instead of `anyhow::Result<()>` ?

Reply:
or `use anyhow::Result as AnyhowResult`

Reply:
@LagginTimes please change back to `anyhow::Result` for all files.